akka-actor-tests/src/test/scala/akka/event/LoggingReceiveSpec.scala

```diff
@@ -229,10 +230,9 @@ class LoggingReceiveSpec extends WordSpec with BeforeAndAfterAll {
           val sname = supervisor.path.toString
           val sclass = classOf[TestLogActor]
 
-          expectMsgAllPF(messages = 3) {
+          expectMsgAllPF(messages = 2) {
             case Logging.Debug(`sname`, `sclass`, msg: String) if msg startsWith "started" ⇒ 0
             case Logging.Debug(_, _, msg: String) if msg startsWith "now supervising"      ⇒ 1
-            case Logging.Debug(_, _, msg: String) if msg startsWith "now watched by"       ⇒ 2
```

Comment:
Hmm if it's a carry-over from another test we definitely don't want it here indeed - I assumed it was a 'side effect' of something else in this test and added it to be more explicit. So good to remove it :).